RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -338,6 +338,11 @@ boolean handleFamiliar(familiar fam)
 		fam = $familiar[Ms. Puck Man];
 	}
 	
+	if(!canChangeToFamiliar(fam))
+	{
+		return false;
+	}
```

Comment:
if we do that. we can probably remove most of the other checks in this function.

Reply:
yes. I was confused by return true but that can't matter if the familiar can't be changed to

Reply:
maybe not the line with !pathAllowsChangingFamiliar(). handleFamiliar returns false for that
canChangeToFamiliar(fam) returns true for current familiar in_quantumTerrarium() before it returns false for !pathAllowsChangingFamiliar(), so it's not strictly the same check